modules/statistics/server/controllers/statistics.server.controller.js

```diff
@@ -25,109 +121,76 @@ exports.get = function(req, res) {
 
     // Total users
     function(done) {
-      User
-        .find({ public: true })
-        .count(function(err, count) {
-          req.statistics.total = count;
-          done(err);
-        });
+      exports.getUsersCount(function(err, count) {
+        if (err) {
+          return done(err);
+        }
+        req.statistics.total = count;
+        done();
+      });
     },
 
     // External sites - BeWelcome
     function(done) {
-
-      User.count({
-        extSitesBW: { $exists: true, $ne: '' },
-        public: true
-      }, function(err, count) {
-        req.statistics.connected.bewelcome = count || 0;
+      exports.getExternalSiteCount('bewelcome', function(err, count) {
+        req.statistics.connected.bewelcome = count;
         done(err);
```

Comment:
Here we're setting the `count` even if we got an error. I think it'd probably be more appropriate to call `done(err)` if there is an `err` and then set the count, and then call `done()`, like lines 125-130 above.